core/src/world/place/building/religious/shrine.rs

```diff
@@ -0,0 +1,238 @@
+    if last_char == 'e' {
+        format!("{}ing", &verb[..verb.len() - 1])
+    } else {
+        format!("{}ing", verb)
+    }
+}
+
+fn pluralize(noun: String) -> String {
+    let last_char = noun.chars().last().unwrap();
+    let last_two_chars = &noun[noun.len() - 2..noun.len()];
+    if last_char == 'y' && !vec!['a', 'e', 'i', 'o', 'u'].contains(&noun.chars().nth(noun.len() - 2).unwrap()) {
+        format!("{}ies", &noun[..noun.len() - 1])
+    } else if last_two_chars == "ch" || last_two_chars == "sh" {
+        format!("{}es", noun)
+    } else if last_char == 's' || last_char == 'x' || last_char == 'z' {
+        format!("{}es", noun)
+    } else if last_char == 'f'{
+        format!("{}ves",&noun[..noun.len() -1])
+    }else {
+        format!("{}s", noun)
+    }
+}
+
+fn number(rng: &mut impl Rng) -> &'static str {
+    #[rustfmt::skip]
+    const NUMBERS: &[&str] = 
+        &[
+            "Two","Three","Four","Five","Six","Seven","Eight","Eight and a Half","Nine",
+            "Twelve","Thirty-Six", "Forty","Seventy-Two","Nine and Twenty", "Ninety-Nine","Thousand"
+        ];
+    NUMBERS[rng.gen_range(0..NUMBERS.len())]
+}
+
+//NOUN
+fn noun(rng: &mut impl Rng) -> String {
+    #[rustfmt::skip]
+    const NOUNS: &[&str] = &[
+        "Blight","Death","Ghost","Honor", "Mirror","Omen","Oracle","Pearl",
+        "Phantom","Pheonix","Spirit","Soul","Shadow","Blood","Dream","Emerald","Iron"
+    ];
+
+    NOUNS[rng.gen_range(0..NOUNS.len())].to_string()
+}
+//DEITY can be PERSON, ANIMAL, or DIVINE CONCEPT
+fn deity(rng: &mut impl Rng) -> String {
+    match rng.gen_range(0..6) {
+        0..=1 => format!("The {}",person(rng)),
+        2     => format!("The {} {}",descriptor(rng),person(rng)),
+        3..=4 => format!("The {}",animal(rng)),
+        5 => format!("The {} {}",descriptor(rng),animal(rng)),
+        6..=8 => format!("{}",concept(rng)),
+        9 => format!("{} {}",descriptor(rng),concept(rng)),
+        _ => unreachable!(),
+    }
+}
+//PERSON
+fn person(rng: &mut impl Rng) -> &'static str {
+    #[rustfmt::skip]
+    const PEOPLE: &[&str] = &[
+        "Father","Mother","Parent","Sibling","Hunter","Emperor","Empress","Ruler","Warrior","Sage"
+    ];
+    PEOPLE[rng.gen_range(0..PEOPLE.len())]
+}
+//ANIMAL
+fn animal(rng: &mut impl Rng) -> String {
+    #[rustfmt::skip]
+    const ANIMALS: &[&str] = &[
+        "Bear","Beetle","Carp","Cat","Cormorant","Cow","Crab","Deer","Dog","Fox",
+        "Frog","Goat","Hart","Hawk","Heron","Horse",
+        "Hound","Lion","Magpie","Owl","Panther","Peacock","Phoenix",
+        "Rabbit","Ram","Rat","Raven","Salamander","Scorpion","Rat","Rabbit",
+        "Snake","Spider","Squid","Squirrel","Stag","Tiger","Toad","Tortoise","Turtle",
+        "Unicorn", "Vulture", "Wolf",
+    ];
+    ANIMALS[rng.gen_range(0..ANIMALS.len())].to_string()
+}
+//DIVINE CONCEPT
+fn concept(rng: &mut impl Rng) -> &'static str {
+    const CONCEPTS: &[&str] = &[
```

Comment:
```suggestion
    #[rustfmt::skip]
    const CONCEPTS: &[&str] = &[
```

Adding this directive will prevent `rustfmt` from splatting all of your vocabulary onto separate lines.